preferences.py

```diff
@@ -123,7 +123,7 @@ class model_lookup:
 
     @classmethod
     def get(cls, item):
-        return cls._models.get(item, item)
+        return cls._models.get(item, None)
```

Comment:
We don't want to default to returning the model id string in case if the id is invalid or the models haven't loaded yet.